packages/app/src/components/Live/Live.tsx

```diff
@@ -28,28 +29,36 @@ export const Live = ({
         `,
         me
     );
+
+    const [overlayVisible, setOverlayVisible] = useState(false);
+
     return (
         <SafeAreaView
-            style={{ flex: 1, backgroundColor: '#d2ffff', height: '100%' }}
+            style={{
+                flex: 1,
+                backgroundColor: 'black'
+            }}
         >
-            <View style={{ flex: 1 }}>
-                <Video
-                    source={{
-                        uri:
-                            'https://fra-cdn.livepeer.com/hls/8197mqr3gsrpeq37/index.m3u8'
-                    }}
-                    ref={player}
-                    // onBuffer={this.onBuffer} // Callback when remote video is buffering
-                    // onError={this.videoError} // Callback when video cannot be loaded
+            <View style={{ flex: 1, justifyContent: 'space-between' }}>
+                <LiveVideo />
+
+                <TouchableOpacity
+                    onPressOut={() => setOverlayVisible(!overlayVisible)}
```

Comment:
turns out that `onPress` is the best here. i tested it and it fixes all those weird  interactions